src/ngMock/angular-mocks.js

```diff
@@ -193,6 +239,72 @@ angular.mock.$Browser.prototype = {
   }
 };
 
+/**
+ * @ngdoc function
+ * @name $flushPendingTasks
+ *
+ * @description
+ * Flushes all pending tasks and executes the corresponding callbacks.
+ *
+ * The types of tasks that are flushed include:
+ *
+ * - Pending timeouts (via {@link $timeout}).
+ * - Pending tasks scheduled via {@link ng.$rootScope.Scope#$applyAsync}.
```

Comment:
This must be `{@link ng.$rootScope.Scope#$applyAsync $applyAsync}` or similar, otherwise it'll show `$rootScope.Scope` as the name. 

Same in the line below.